django/river/settings/base.py

```diff
@@ -0,0 +1,216 @@
+    "django.middleware.clickjacking.XFrameOptionsMiddleware",
+]
+
+ROOT_URLCONF = "river.urls"
+
+TEMPLATES = [
+    {
+        "BACKEND": "django.template.backends.django.DjangoTemplates",
+        "DIRS": [],
+        "APP_DIRS": True,
+        "OPTIONS": {
+            "context_processors": [
+                "django.template.context_processors.debug",
+                "django.template.context_processors.request",
+                "django.contrib.auth.context_processors.auth",
+                "django.contrib.messages.context_processors.messages",
+            ],
+        },
+    },
+]
+
+WSGI_APPLICATION = "river.wsgi.application"
+
+
+# Database
+# https://docs.djangoproject.com/en/3.1/ref/settings/#databases
+
+# Not actually use for anything yet (other than sessions, etc.)
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.sqlite3",
+        "NAME": BASE_DIR / "db.sqlite3",
+    }
+}
+
+
+# Password validation
+# https://docs.djangoproject.com/en/3.1/ref/settings/#auth-password-validators
+
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator",
+    },
+    {
+        "NAME": "django.contrib.auth.password_validation.MinimumLengthValidator",
+    },
+    {
+        "NAME": "django.contrib.auth.password_validation.CommonPasswordValidator",
+    },
+    {
+        "NAME": "django.contrib.auth.password_validation.NumericPasswordValidator",
+    },
+]
+
+
+# Internationalization
+# https://docs.djangoproject.com/en/3.1/topics/i18n/
+
+LANGUAGE_CODE = "en-us"
+
+TIME_ZONE = "Europe/Paris"
+
+USE_I18N = True
+
+USE_L10N = True
+
+USE_TZ = True
+
+
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/3.1/howto/static-files/
+
+STATIC_URL = "/static/"
+STATIC_ROOT = Path(os.environ.get("STATIC_ROOT", "var/www/static"))
+
+
+# Logging
+# https://docs.djangoproject.com/en/3.1/topics/logging/
+
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "default": {
+            "format": "{asctime} [{levelname:>8}] {filename:>15}:{lineno:3d} {message}",
+            "style": "{",
+        },
+        "fluentd": {
+            "()": "utils.log.FluentFormatter",
+            "format": {
+                "logger": "{name}",
+                "where": "{module}.{funcName}",
+                "level": "{levelname}",
+                "host": "{hostname}",
+            },
+            "style": "{",
+        },
+    },
+    "handlers": {
+        "console": {
+            "class": "logging.StreamHandler",
+            "formatter": "default",
+        },
+        "fluentd": {
+            "class": "fluent.handler.FluentHandler",
+            "formatter": "fluentd",
+            "tag": "river",
+            "host": os.environ.get("FLUENTD_HOST", "fluentd"),
+            "port": int(os.environ.get("FLUENTD_PORT", 24224)),
+        },
+    },
+    "root": {
+        "handlers": ["console", "fluentd"],
```

Comment:
so this means we always log to the console ? don't we want to do that only in debug mode ?

Reply:
We always want to log to the console. The only potential issue would be the growing size of the logs, but then we should have a log retention policy. What do you think ?

Reply:
I agree on the retention policy, do you think we could have a shorter one for the console than for fluentd ? It's something we will have to deal with shortly since we already have storage space issues at Reims (but we can still remove the container to free some space in the meantime)